README.md

```diff
@@ -63,23 +67,24 @@ Kubernetes version numbers.
 
 #### Branches and tags.
 
-We will create a new branch and tag for each increment in the major version number or
-minor version number. We will create only a new tag for each increment in the patch
+We will create a new branch and tag for each increment in the minor version
+number. We will create only a new tag for each increment in the patch
 version number. See [semver](http://semver.org/) for definitions of major,
 minor, and patch.
 
 The master branch will track HEAD in the main Kubernetes repo and
-accumulate changes. Consider HEAD to have the version `x.(y+1).0-alpha` or
-`(x+1).0.0-alpha` (depending on whether it has accumulated a breaking change or
-not), where `x` and `y` are the current major and minor versions.
+accumulate changes. Consider HEAD to have the version `x.(y+1).0-alpha`,
+where `x` and `y` are the current major and minor versions.
 
 #### Compatibility: your code <-> client-go
 
-`client-go` follows [semver](http://semver.org/), so until the major version of
-client-go gets increased, your code will compile and will continue to work with
-explicitly supported versions of Kubernetes clusters. You must use a dependency
-management system and pin a specific major version of `client-go` to get this
-benefit, as HEAD follows the upstream Kubernetes repo.
+`client-go` follows [semver](http://semver.org/)-like tags, so until the minor
+version of client-go gets increased, your code will compile and will continue
+to work with explicitly supported versions of Kubernetes clusters.
+
+You must use a dependency management system and pin a specific major version of
```

Comment:
Drop most of this paragraph, major versions do not apply to client-go since all our releases are under the same v0 major version. Just leave the reference to INSTALL.md for guidelines on requiring a specific version of client-go